assets/js/theme/common/product-details.js

```diff
@@ -443,6 +443,38 @@ export default class ProductDetails extends ProductDetailsBase {
         this.setLiveRegionAttributes($addToCartBtn.next(), 'status', 'polite');
     }
 
+    /**
+     *
+     * Add a product to wishlist
```

Comment:
could we have more descriptive name here?

Reply:
@BC-tymurbiedukhin done!